lib/travis/cli/api_command.rb

```diff
@@ -108,7 +108,7 @@ def setup_enterprise
             endpoint
           end
           self.api_endpoint             = c[enterprise_name]
-          self.insecure                 = true if insecure.nil?
+          self.insecure                 = true unless insecure.nil?
```

Comment:
@shawnzhu - Sorry for the delay! Ok, looking closer I do think this is fine. But I want to get a second opinion since I'm not super familiar with the code base. I'm going to reach out  to other engineers and see if they can take a look at the whole change!